### ISSUE TITLE
Log Out doesn't log out

### ISSUE THREAD

**Issue by Ricardo-Simoes** [sindresorhus/caprine#772]
I'm using Caprine (AppImage) version 2.28.0 and the "Log Out" option on the "File" menu, doesn't do nothing... can anyone please confirm? I'm using Manjaro Linux XFCE x64.

**Comment by whitecrownclown** [comment]
@Ricardo-Simoes It works for me. I'm on 2.28.0 on macos 10.14.3. Could you provide more info on this? Try opening the console and see if there's an error (shortcut should be F12).

**Comment by Ricardo-Simoes** [comment]
@whitecrownclown I have been able to detect what was causing the "Log Out" problem. It seems there was some leftovers on my ~/.config/Caprine folder from the previous AppImage file. I particularly noticed something happening with the "Cookies" file, so what I deleted the entire ~/.config/Caprine folder, executed the caprine AppImage file again, it recreated the folder and everything is now normal. What I learned from this, is that uninstalling an older version of the Caprine AppImage file just by deleting it, may not be sufficient because when you download a more recent version and run it, it still uses the old ~/.config/Caprine files, and that could lead to something not working right.
